<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsintModulesWidget</class>
 <widget class="QWidget" name="OsintModulesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="useProfiles">
     <property name="text">
      <string>Use Profiles</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameProfiles">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QComboBox" name="comboBoxProfiles">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonLoadProfile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDeleteProfile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLineEdit" name="lineEditNewProfile">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCreateProfile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Free Osint Engines</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_noKeyEngines">
         <property name="cursor" stdset="0">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-210</y>
            <width>162</width>
            <height>910</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Certificates</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleCrtsh">
             <property name="text">
              <string>Crtsh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleCensysFree">
             <property name="text">
              <string>Censys</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleCertspotter">
             <property name="text">
              <string>CertSpotter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleGoogleCert">
             <property name="text">
              <string>GoogleCert</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>APIs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleAnubis">
             <property name="text">
              <string>Anubis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleBgpview">
             <property name="text">
              <string>Bgpview</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleDnsbufferoverrun">
             <property name="text">
              <string>DnsBufferOverRun</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleHackerTargetFree">
             <property name="text">
              <string>HackerTarget (Free)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleMnemonicFree">
             <property name="text">
              <string>Mnemonic (Free)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleOmnisint">
             <property name="text">
              <string>Omnisint</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleOtxFree">
             <property name="text">
              <string>Otx (Free)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleRobtexFree">
             <property name="text">
              <string>Robtex (Free)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleSublist3r">
             <property name="text">
              <string>sublist3r</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleThreatcrowd">
             <property name="text">
              <string>ThreatCrowd</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleThreatminer">
             <property name="text">
              <string>ThreatMiner</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleUrlscan">
             <property name="text">
              <string>urlscan</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Sites</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleDnsdumpster">
             <property name="text">
              <string>DnsDumpster</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleNetcraft">
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="text">
              <string>Netcraft</string>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="modulePagesInventory">
             <property name="text">
              <string>PagesInventory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="modulePkey">
             <property name="text">
              <string>pKey</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleRapiddns">
             <property name="text">
              <string>RapidDns</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleSuip">
             <property name="text">
              <string>Suip</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleSitedossier">
             <property name="text">
              <string>SiteDossier</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Scrappers</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleAsk">
             <property name="text">
              <string>Ask</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleBaidu">
             <property name="text">
              <string>Baidu</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleBing">
             <property name="text">
              <string>Bing</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleDogpile">
             <property name="text">
              <string>DogPile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleDuckduckgo">
             <property name="text">
              <string>DuckduckGo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleExalead">
             <property name="text">
              <string>Exalead</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleTrello">
             <property name="text">
              <string>Trello</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleYahoo">
             <property name="text">
              <string>Yahoo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Archives</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleCommonCrawl">
             <property name="text">
              <string>CommonCrawl</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleWaybackmachine">
             <property name="text">
              <string>WayBackMachine</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleArchiveit">
             <property name="text">
              <string>ArchiveIt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleArchiveToday">
             <property name="text">
              <string>ArchiveToday</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Paid Osint Engines</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_keyEngines">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>161</width>
            <height>677</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QCheckBox" name="moduleBinaryEdge">
             <property name="text">
              <string>BinaryEdge</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleC99">
             <property name="text">
              <string>C99</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleCircl">
             <property name="text">
              <string>Circl</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleGithub">
             <property name="text">
              <string>Github</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleHackerTargetPaid">
             <property name="text">
              <string>HackerTarget</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleHuntersearch">
             <property name="text">
              <string>HunterSearch</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleIpApi">
             <property name="text">
              <string>IpApi</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleIpInfo">
             <property name="text">
              <string>IpInfo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleMnemonicPaid">
             <property name="text">
              <string>Mnemonic</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleSpyse">
             <property name="text">
              <string>spyse</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleVirustotalapi">
             <property name="text">
              <string>VirusTotal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleOtxPaid">
             <property name="text">
              <string>Otx</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleProjectdiscovery">
             <property name="text">
              <string>ProjectDiscovery</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleRiskIq">
             <property name="text">
              <string>RiskIq</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleRobtexPaid">
             <property name="text">
              <string>Robtex</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleSecuritytrails">
             <property name="text">
              <string>SecurityTrails</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleShodan">
             <property name="text">
              <string>Shodan</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleThreatBook">
             <property name="text">
              <string>ThreatBook</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleViewDns">
             <property name="text">
              <string>ViewDns</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleWebResolver">
             <property name="text">
              <string>WebResolver</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleWhoisXmlApi">
             <property name="text">
              <string>WhoisXmlApi</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleZetalytics">
             <property name="text">
              <string>ZETAlytics</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleZoomeye">
             <property name="text">
              <string>ZoomEye</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleBingPaid">
             <property name="text">
              <string>Bing</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleCensys">
             <property name="text">
              <string>Censys</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleCloudflare">
             <property name="text">
              <string>CloudFlare</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleGoogle">
             <property name="text">
              <string>Google</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="moduleIntelx">
             <property name="text">
              <string>IntelX</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="modulePentesttools">
             <property name="text">
              <string>pentest-tools</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>89</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
